restapi/admin_users_test.go

```diff
@@ -32,6 +32,8 @@ import (
 var minioListUsersMock func() (map[string]madmin.UserInfo, error)
 var minioAddUserMock func(accessKey, secreyKey string) error
 var minioRemoveUserMock func(accessKey string) error
+var minioUpdateUserGroupsMock func(user string, groupsToAssign []string) (*madmin.UserInfo, error)
+var minioGetUserInfo func(accessKey string) (madmin.UserInfo, error)
```

Comment:
```suggestion
var minioGetUserInfoMock func(accessKey string) (madmin.UserInfo, error)
```
To keep consistency we need to call it mock to know it is part of testing

Reply:
Updated function name